### DISCUSSION TITLE
Zotero的条目信息改变

### DISCUSSION THREAD

**Discussion by lisch7** [zotero-chinese/wiki#316] [求助 / Q&A]
这是一个困扰我很久的问题，我是一个多端使用的用户（3台电脑+1台ipad），使用坚果云web dev进行同步。我很长一段时间都收到这样一个困扰，就是我Zotero仓库某些条目的信息会神奇地发生变化，例如，这个条目本来的期刊名是BMJ，然后我某一天突然发现它的简称变成了immunity，我确定所有的简称本来都应该是BMJ的，类似这个条目，新建的时间是2024-9-13，但是修改的时间是2024-10-29，而这个10-29可能是我打开另一台设备进行同步的时间：
![image](https://github.com/user-attachments/assets/25de17f0-7970-4e11-a656-1c76f03f7954)

这个现象发生了很多次，以至于插入参考文献的时候，我都害怕我会插入错误的文献信息。所以我每次插入的时候都要重新更新元数据确保数据正确，这给我带来了很大的负担。这个问题我也去Zotero的官方讨论区提交过，但是人家说从来没碰到过这样的问题

不知道有没有高手知道这个问题是怎么造成的，有没有什么好的方式解决？谢谢！

**Comment by zepinglee** [comment]
估计是插件错误地设置了简称。你都用了哪些插件？

有的插件在抓取条目时填写刊名简称字段，但是其使用的数据可能有误。另外如果你在设备 A 上添加了新的文献，同步到设备 B 时，有的插件可能会当成新抓取的文献，所以执行上述填写刊名简称的过程。

**Reply by lisch7** [reply]
> 估计是插件错误地设置了简称。你都用了哪些插件？
> 
> 有的插件在抓取条目时填写刊名简称字段，但是其使用的数据可能有误。另外如果你在设备 A 上添加了新的文献，同步到设备 B 时，有的插件可能会当成新抓取的文献，所以执行上述填写刊名简称的过程。

我的插件很简单，Jasminum，Zotero One，Zoplicate, Easier Citation。其中只有Zotero One有这个抓取简称字段的功能，但是我看人家都没有这个问题，同时我也把这个自动更新条目的功能关闭了，但是还是有这个问题
